tests/records_test.py

```diff
@@ -501,7 +501,7 @@ def test_get_updated_records_since_date(app):
 
 
 def test_get_updated_records_on_date(app):
-    assert(get_inspire_records_updated_on("2020-06-29") == ['1650066'])
+    assert(type(get_inspire_records_updated_on("2021-06-27")) == list)
```

Comment:
This is a pretty weak test now. Could you use [requests mock](https://requests-mock.readthedocs.io/en/latest/mocker.html#real-http-requests) with `real_http=True` to check what calls are being made?

Reply:
Thanks for the suggestion.  I've changed the test to use `requests_mock` to mock the INSPIRE-HEP response.